response_operations_ui/exceptions/error_codes.py

```diff
@@ -24,5 +27,6 @@ def get_error_code_message(error_code):
         ErrorCode.API_UNEXPECTED_CONTENT_TYPE: "The service returned an unexpected content type",
         ErrorCode.API_UNEXPECTED_CONTENT: "The service returned unexpected content",
         ErrorCode.API_OIDC_CREDENTIALS_ERROR: "An error occurred preparing to authenticate with the service",
+        ErrorCode.NOT_FOUND: "Page Not Found",
```

Comment:
Again this should probably follow the pattern `API_NOT_FOUND` and `Resource Not Found` for this use case. We can update again.